tutorials/misc/plot_ecog.py

```diff
@@ -11,102 +11,81 @@
+- ECoG_ data from a study of epilepsy subject with a seizure
+- channel locations in FreeSurfer's ``fsaverage`` MRI space
+- projection onto a pial surface
 
 For a complementary example that involves sEEG data, channel locations in
 MNI space, or projection into a volume, see :ref:`tut_working_with_seeg`.
+
+.. _ECoG: https://openneuro.org/datasets/ds003029
+
 """
 # Authors: Eric Larson <larson.eric.d@gmail.com>
 #          Chris Holdgraf <choldgraf@gmail.com>
 #          Adam Li <adam2392@gmail.com>
+#          Alex Rockhill <aprockhill@mailbox.org>
+#          Liberty Hamilton <libertyhamilton@gmail.com>
 #
 # License: BSD (3-clause)
 
-import pandas as pd
+import os.path as op
+
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.animation as animation
+from mne_bids import BIDSPath, read_raw_bids
 
 import mne
 from mne.viz import plot_alignment, snapshot_brain_montage
 
 print(__doc__)
 
 # paths to mne datasets - sample ECoG and FreeSurfer subject
-misc_path = mne.datasets.misc.data_path()
+bids_root = mne.datasets.epilepsy_ecog.data_path()
 sample_path = mne.datasets.sample.data_path()
-subject = 'sample'
-subjects_dir = sample_path + '/subjects'
+subjects_dir = op.join(sample_path, 'subjects')
 
-###############################################################################
-# Let's load some ECoG electrode locations and names, and turn them into
-# a :class:`mne.channels.DigMontage` class. First, use pandas to read in the
-# ``.tsv`` file.
-
-# In this tutorial, the electrode coordinates are assumed to be in meters
-elec_df = pd.read_csv(misc_path + '/ecog/sample_ecog_electrodes.tsv',
-                      sep='\t', header=0, index_col=None)
-ch_names = elec_df['name'].tolist()
-ch_coords = elec_df[['x', 'y', 'z']].to_numpy(dtype=float)
-ch_pos = dict(zip(ch_names, ch_coords))
-# Ideally the nasion/LPA/RPA will also be present from the digitization, here
-# we use fiducials estimated from the subject's FreeSurfer MNI transformation:
-lpa, nasion, rpa = mne.coreg.get_mni_fiducials(
-    subject, subjects_dir=subjects_dir)
-lpa, nasion, rpa = lpa['r'], nasion['r'], rpa['r']
 
 ###############################################################################
-# Now we make a :class:`mne.channels.DigMontage` stating that the ECoG
-# contacts are in the FreeSurfer surface RAS (i.e., MRI) coordinate system.
+# Let's load some ECoG electrode data with ``mne-bids``.
+# first define the bids path
+bids_path = BIDSPath(root=bids_root, subject='pt1', session='presurgery',
+                     task='ictal', datatype='ieeg', extension='vhdr')
 
-montage = mne.channels.make_dig_montage(
-    ch_pos, coord_frame='mri', nasion=nasion, lpa=lpa, rpa=rpa)
-print('Created %s channel positions' % len(ch_names))
+# then we'll use it to load in the sample dataset
+# Here we use a format (iEEG) that is only available in MNE-BIDS 0.7+, so it
+# will emit a warning on versions <= 0.6
+raw = read_raw_bids(bids_path=bids_path, verbose=False)
```

Comment:
to get it to run locally I still need
```py
with warnings.catch_warnings(record=True):
    warnings.simplefilter("ignore")
    raw = read_raw_bids(bids_path=bids_path, verbose=False)
```
but @larsoner's changes should make it work on CircleCI now